src/cider-velox/benchmark/expression/CiderComparisonConjunct.cpp

```diff
@@ -19,35 +19,36 @@
  * under the License.
  */
 
+// #define _LOGGING_H_  // ignore glog
```

Comment:
remove this line if it's not needed anymore.

Reply:
done